ar_exercise/exercise.rb

```diff
@@ -51,21 +72,43 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+abc = apartments.new(address: "1234 north", monthly_rent: 800,sqft: 1200,num_beds:2,num_baths:1 )
+abc.save
+defg = apartments.new(address: "5678 West", monthly_rent: 950,sqft: 900,num_beds:1,num_baths:1 )
+defg.save
+hij = apartments.new(address: "9101 East", monthly_rent: 1000,sqft: 1300,num_beds:2,num_baths:2 )
+hij.save
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
+aaaa = Artist.create(name: "aaaa bbbb", age: 22, gender: "Male", apartment_id: 18)
+cccc = Artist.create(name: "cccc dddd", age: 32, gender: "Female", apartment_id: 16)
+eeee = Artist.create(name: "eeee ffff", age: 42, gender: "Male", apartment_id: 20)
+gggg = Artist.create(name: "gggg hhhh", age: 21, gender: "Male", apartment_id: 8)
+iiii = Artist.create(name: "iiii  jjjj" age: 31, gender: "Female", apartment_id: 9)
+kkkk = Artist.create(name: "kkkk llll", age: 25, gender: "Male", apartment_id: 11)
+mmmm = Artist.create(name: "mmmm nnnn", age: 28, gender: "Male", apartment_id: 15)
+oooo = Artist.create(name: "oooo pppp", age: 62, gender: "Female", apartment_id: 16)
+qqqq = Artist.create(name: "qqqq rrrr", age: 72, gender: "Male", apartment_id: 19)
+
+
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
-
+Kristin=tenants.find_by(name: "Kristin Wisoky")
+Kristin.Update(age: age+1)
```

Comment:
- You should not capitalize a variable unless it is a class
- This is going to look for a local variable `age`, which at the moment does not exist
- `.update` should begin with a lowercase u